frontend/src/components/processManagement/ProcessManageBoard/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Process } from '@customTypes/process';
+import { Fragment } from 'react/jsx-runtime';
+import S from './style';
+import ProcessModifyForm from '../ProcessModifyForm';
+import ProcessAddButton from '../ProcessAddButton';
+
+interface ProcessManageBoardProps {
+  processes: Process[];
+}
+
+export default function ProcessManageBoard({ processes }: ProcessManageBoardProps) {
+  const FIRST_INDEX = 0;
+  const LAST_INDEX = processes.length - 1;
+  const isAddable = processes.length !== 5;
```

Comment:
정책상 관리 가능한 프로세스가 최대 5개 이하라면 그 의미가 잘 반영되도록 부등호를 사용해주시면 좋을 것 같아요!
```suggestion
  const isAddable = processes.length < 5;
```

Reply:
[커밋 내용](https://github.com/woowacourse-teams/2024-cruru/pull/161/commits/3f6bc095d00ef030d21ddbbe97503246ce129b58) 확인했습니다. 감사해요!